From 76083a7387c60a197e0a159bd3ae094d4dea30ea Mon Sep 17 00:00:00 2001
From: Robin Getz <robin.getz@analog.com>
Date: Tue, 16 Jul 2019 17:54:53 -0400
Subject: [PATCH 06/12] travis-ci: recent upgrades in the travis-ci
 infrastructure caused some breakages in deployments

Requires adding -oHostKeyAlgorithms=+ssh-dss on a few more distributions

Signed-off-by: Robin Getz <robin.getz@analog.com>
---
 .travis.yml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.travis.yml b/.travis.yml
index 078807d..8e4871c 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -59,14 +59,17 @@ matrix:
       env:
         - OS_TYPE=centos_docker
         - OS_VERSION=6
+        - EXTRA_SSH=-oHostKeyAlgorithms=+ssh-dss
     - os: linux
       env:
         - OS_TYPE=centos_docker
         - OS_VERSION=7
+        - EXTRA_SSH=-oHostKeyAlgorithms=+ssh-dss
     - os: linux
       env:
         - OS_TYPE=ubuntu_docker
         - OS_VERSION=bionic
+        - EXTRA_SSH=-oHostKeyAlgorithms=+ssh-dss
     - compiler: "gcc"
       os: osx
       osx_image: xcode9.2
-- 
2.20.1

